nservicebus/usage/index.md

```diff
@@ -26,15 +26,18 @@ The recommended method for measuring usage in an NServiceBus system is via Servi
 
 This method requires ServicePulse version 1.40 or later, and ServiceControl version 5.4 or later. Additionally, if using RabbitMQ broker, version 3.10.0 or higher is required.
 
-To learn more about this method, read the [ServicePulse Usage documentation](/servicepulse/usage.md). 
+To learn more about this method, read the [ServicePulse Usage documentation](/servicepulse/usage.md).
+
+> [!NOTE]
+> While customers are not required to use ServicePulse at this time, it will become a requirement in the future.
```

Comment:
```suggestion
> While customers are not required to use ServicePulse at this time, it will become a requirement in all none maintenance-only projects in the future.
```

Reply:
Just a suggestion, I want to make it clear that if a project isn't being updated we will be ok with that

Reply:
What about this so it reads better:

```suggestion
> While customers are not required to use ServicePulse at this time, it will become a requirement for all active projects in the future.
```

Reply:
Sounds better, yes thanks !